src/Components/FeaturedReviews.jsx

```diff
@@ -15,19 +15,32 @@ export const FeaturedReviews = ({ reviews }) => {
         <h2>Featured Reviews</h2>
       </header>
       <main>
+        <section>
+          <Link to="/reviews" className="link-to-page">
+            See All Reviews
+          </Link>
+        </section>
+
         <ul className="reviews">
           {indexArr.map((index) => {
             return (
               <li
                 key={`featured-${reviews[index].review_id}`}
                 className="review-card"
               >
-                <h3>{reviews[index].title}</h3>
-                <img
-                  src={reviews[index].review_img_url}
-                  alt={reviews[index].title}
-                  className="review-img"
-                />
+                <section>
+                  <Link
+                    to={`/reviews/${reviews[index].review_id}`}
+                    id="review-title"
```

Comment:
id should be unique, className can be used for CSS if multiple tags require same styling